test_integration/lib/config/test_names.dart

```diff
@@ -3,51 +3,57 @@
 
-  static const String realtimePublish = 'realtimePublish';
+  // realtime
   static const String realtimeEncryptedPublish = 'realtimeEncryptedPublish';
-  static const String realtimePublishSpec = 'realtimePublishSpec';
   static const String realtimeEncryptedPublishSpec =
       'realtimeEncryptedPublishSpec';
   static const String realtimeEvents = 'realtimeEvents';
-  static const String realtimeSubscribe = 'realtimeSubscribe';
   static const String realtimeHistory = 'realtimeHistory';
-  static const String realtimeTime = 'realtimeTime';
-  static const String realtimePublishWithAuthCallback =
-      'realtimePublishWithAuthCallback';
-  static const String realtimePresenceGet = 'realtimePresenceGet';
-  static const String realtimePresenceHistory = 'realtimePresenceHistory';
+  static const String realtimeHistoryWithAuthCallback =
+      'realtimeHistoryWithAuthCallback';
   static const String realtimePresenceEnterUpdateLeave =
       'realtimePresenceEnterUpdateLeave';
+  static const String realtimePresenceGet = 'realtimePresenceGet';
+  static const String realtimePresenceHistory = 'realtimePresenceHistory';
   static const String realtimePresenceSubscribe = 'realtimePresenceSubscribe';
-// TODO(tiholic) handle realtimeHistoryWithAuthCallback
+  static const String realtimePublish = 'realtimePublish';
+  static const String realtimePublishSpec = 'realtimePublishSpec';
+  static const String realtimePublishWithAuthCallback =
+      'realtimePublishWithAuthCallback';
+  static const String realtimeSubscribe = 'realtimeSubscribe';
+  static const String realtimeTime = 'realtimeTime';
 
-  static const String cryptoGenerateRandomKey = 'cryptoGenerateRandomKey';
-  static const String cryptoEnsureSupportedKeyLength =
-      'cryptoEnsureSupportedKeyLength';
-  static const String cryptoGetDefaultParams = 'cryptoGetDefaultParams';
+  // rest
+  static const String restCapabilities = 'restCapabilities';
+  static const String restEncryptedPublish = 'restEncryptedPublish';
+  static const String restEncryptedPublishSpec = 'restEncryptedPublishSpec';
+  static const String restHistory = 'restHistory';
+  static const String restHistoryWithAuthCallback =
+      'restHistoryWithAuthCallback';
+  static const String restPresenceGet = 'restPresenceGet';
+  static const String restPresenceHistory = 'restPresenceHistory';
+  static const String restPublish = 'restPublish';
+  static const String restPublishSpec = 'restPublishSpec';
+  static const String restPublishWithAuthCallback =
+      'restPublishWithAuthCallback';
+  static const String restTime = 'restTime';
+
+  // helpers
+  static const String testHelperFlutterErrorTest = 'testHelperFlutterErrorTest';
+  static const String testHelperUnhandledExceptionTest =
+      'testHelperUnhandledExceptionTest';
 
-  // This is not a test, but a way to retrieve
   // more information of failures from any of the tests cases
+  // This is not a test, but a way to retrieve
```

Comment:
Now the comment lines are in the inverted order :sweat_smile: Please revert it to how it was before :wink: